packages/cli/src/commands/index.ts

```diff
@@ -214,6 +214,7 @@ export function registerCommands(program: CommanderStatic) {
 
   program
     .command('versions:bump')
+    .option('--prefix <glob>', 'Override glob for matching packages to upgrade')
```

Comment:
I think `--pattern` fits a bit better here right? since that's really what we're providing now

Reply:
Yep you are right will update 👍